.github/workflows/build.yml

```diff
@@ -60,10 +60,10 @@ jobs:
           ~/.cargo/registry/cache/
           ~/.cargo/git/db/
           target/
-        key: simd-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', '**/Cargo.toml') }}
+        key: simd-test-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', '**/Cargo.toml') }}
         restore-keys: |
-          simd-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', '**/Cargo.toml') }}
-          simd-${{ runner.os }}-cargo-
+          simd-test-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', '**/Cargo.toml') }}
+          simd-test-${{ runner.os }}-cargo-
```

Comment:
nitpick, could you change the cache key to something like `simd-2-` here?

Reply:
> nitpick, could you change the cache key to something like `simd-2-` here?

I'll fix this.